ember-intl/addon/helpers/format-date.ts

```diff
@@ -53,6 +53,6 @@ export default class FormatDateHelper extends Helper<FormatDateSignature> {
       // }
     }
 
-    return this.intl.formatDate(value, options);
+    return this.intl.formatDate(value!, options);
```

Comment:
**TODO**

Replace `isEmpty()` with `value === undefined || value === null` (target `ember-intl@v7`). The former is ambiguous (allows too many inputs to result in `true`) and doesn't work with TypeScript, as we see from the need to add the non-null assertion operator.